project_task_stage_change_restriction/tests/test_stage_change_restriction.py

```
@@ -0,0 +1,255 @@
+
+from odoo import SUPERUSER_ID
+from odoo.exceptions import UserError
+from odoo.tests import common
+
+
+@common.tagged("-at_install", "post_install")
+class TestStageChangeRestriction(common.TransactionCase):
+    """Validate stage‑change & creation access rules for project tasks."""
+
+    @classmethod
+    def setUpClass(cls):
```

Ruff (0.11.9)

Missing return type annotation for classmethod `setUpClass`

Add return type annotation: `None`

(ANN206)